.dependabot/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 1
+update_configs:
+  # Gradle Dependencies
+  - package_manager: "java:gradle"
+    directory: "/"
+    update_schedule: "daily"
+    automerged_updates:
+      - match:
+          dependency_name: "org.testcontainers:testcontainers-bom"
+      - match:
+          dependency_name: "org.awaitility:awaitility"
+      - match:
+          dependency_name: "org.junit:junit-bom"
+  # Docker Dependencies
+  - package_manager: "docker"
+    directory: "/"
+    update_schedule: "daily"
+  # Plugins Dependencies
```

Comment:
could you please add entries for every plugin that we have?
https://github.com/bsideup/liiklus/tree/master/plugins

I am also thinking whether we should do it in `settings.gradle` and simply list them there, so that it is always in sync?